examples/custom-edi-schema/Build a Custom EDI Schema and Parser.md

```
@@ -0,0 +1,86 @@
+
+## Run the example
+
+`sample.edi` carries the partner extension in its UNH segment (`...:UN:EXT1`):
+
+```bash
+bal run
+```
+
+Expected output — the custom `new_field` is parsed alongside the standard fields:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)